csharp-basics/exercises/Arithmetic/CalculateArea.Test/UnitTest1.cs

```diff
@@ -0,0 +1,85 @@
+using NUnit.Framework;
+using System;
+
+
+namespace CalculateArea.Test
+{
+    
+    
+    public class Tests
+    {
+        private string _expectedMessage;
+        private double _expectedResult;
+
+        [SetUp]
+        public void Setup()
+        {
```

Comment:
Ja setup ir tukš, vai tad to nevar ņemt ārā?

Reply:
Visi tukšie SetUp izņemti